docs/blog/write-the-docs-2016-year-in-review.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: Jan 16, 2016
+   :tags: stats, year in review
+
+Write the Docs 2016 year in review
+==================================
+
+2016 was a wonderful year for Write the Docs.
+This is our fourth year of existence,
+and this is the first year it feels that we have coalesced into a more cohesive community.
+This post will cover a few stats and other tidbits that we feel mark our progress,
+and we hope to continue to report these each year moving forward.
+
+Conferences
+===========
+
+* 400 attendees in Portland
+* 250 attendees in Prague
+
+The :doc:`conferences </conf/index>` continue to be the yearly physical manifestation of our community.
+They are the time each year where hundreds of wonderful folks gather and have a great time,
+while learning so much from each other.
+Proving each year that work can be both fun and informative is one of my greatest pleasures.
+
+We continue to publish :doc:`videos </videos>` freely to the public.
+This is done so that the information can be most widely shared,
+and help people who may not be able to attend the events.
+Write the Docs always strives to be as inclusive as possible,
+and continuing to publish these videos is part of our commitment to openness.
+
+Slack
+=====
+
+* 1,200 members
+
+This is the part of the community that I've been most happy with this year.
+Our :doc:`Slack network </slack>` has really caught on,
+and folks are discussing a wide range of topics every day.
+It really feels like this is where the community is thriving,
+and it becoming the default way of contacting folks inside the Write the Docs community.
+
+Newsletter
+==========
+
+* 2,485 subscribers
+
+The :doc:`newsletter </newsletter>` has been the second initiative that has really taken off this year.
```

Comment:
Looks like I generally say "this year" meaning the year we're talking about. Seems fine.